tools/cli/defines.h

```diff
@@ -21,6 +21,8 @@
 
 #pragma once
 
+#define TDNF_INSTANCE_LOCK_FILE     "/tmp/.tdnf-instace-lockfile"
```

Comment:
Can we stick to conventions put this inside inside `/var/lock/` or `/var/run/lock/`